src/components/Landing.js

```diff
@@ -8,14 +8,20 @@ import { newOrOld, percentageDifference, updateWarning, insertThinSpace } from '
 import { FaTwitterSquare } from 'react-icons/fa';
 import { useEffect } from 'react';
 import DoomBar from './statistics/DoomBar'
+import Vaccines from './Vaccines'
 
 function Landing() {
     useEffect(() => {
```

Comment:
tu nie powinno być drugiego paramka `[]`?

Reply:
Oj to już nie moje, seba coś od routingu robił tutaj. Jak nie ma [] to się zawsze wykonuje chyba, ale nie wiem czy to pożądane.